draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,10 +627,39 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client MAY wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS ClientHello may be altered by the "esnikeys" value
+of an SVCB response {{svcparamkeys-esnikeys}}. CD (Connection Delay)
+is a configurable parameter. The recommended value is 50 milliseconds,
+as per the guidance in {{!HappyEyeballsV2=RFC8305}}.
+
+An SVCB record is consistent with an active or in-progress connection C
+if the following conditions are met:
+
+1. The A or AAAA address for its SvcDomainName matches the destination
+address of C.
+2. The destination port of C matches that specified in the SVCB record.
+
+If an SVCB record is consistent with an active or in-progress
+connection C, the client may continue using C with any information provided
+by the SVCB record. For example, if an SVCB record with a "esnikeys" value and
+"ipv4hint" or "ipv6hint" values {{svcparamkeys-iphints}} arrives, and one of
```

Comment:
Under the current definition, this record is not consistent with this socket.  The IP hint matches, but consistency is defined in terms of the address record, not the IP hint.

I think we should fix this by changing the definition, e.g.
> An SVCB record is consistent with an active or in-progress connection if the client would attempt an equivalent connection when making use of that record.

Reply:
Fixed -- thanks!